test/integration/noderesourcetopology_test.go

```diff
@@ -73,8 +73,8 @@ type nrtTestUserEntry struct {
 	// description contains the test type and tier as described in noderesourcetopology/TESTS.md
 	// and a short description of the test itself
 	description string
-	initCntReq  []map[v1.ResourceName]string
-	cntReq      []map[v1.ResourceName]string
+	initCntReq  []map[string]string
```

Comment:
we should probably squash this commit in the main one, especially considering we are adding this infra in this PR